### PR TITLE
AF-1424: Support SSH keystore with workbench

### PR DESCRIPTION
**Pull request by pefernan**
AF-1275: SSH Keystore backend
AF-1274: SSH Keystore UI
AF-1277: Integrate SSH Keystore with VFS SSH

Adding missing dependencies...

@ge0ffrey  @ederign could you please take a look?

This is part of an ensemble, please merge with:
https://github.com/kiegroup/appformer/pull/492
https://github.com/kiegroup/kie-wb-common/pull/2177
https://github.com/kiegroup/kie-wb-distributions/pull/829
https://github.com/kiegroup/drools-wb/pull/959
https://github.com/kiegroup/jbpm-wb/pull/1215
https://github.com/kiegroup/optaplanner-wb/pull/308
https://github.com/kiegroup/kie-docs/pull/1156

### CODE REVIEW

**Comment by domhanak** [comment]
jenkins retest this

File: optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/pom.xml [hunk 1]
```diff
@@ -164,6 +164,16 @@
...
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-ssh-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-ssh-backend</artifactId>
+      <scope>test</scope>
+    </dependency>
```
Comment (yurloc) [inline optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/pom.xml:176]: @pefernan Pere, I don't think the solver editor backend should depend on UberFire SSH. Is there some non-obvious reason for that?

**Review by yurloc — CHANGES_REQUESTED** [review]

**Comment by yurloc** [comment]
@pefernan Where do I find the SSH keystore UI? I'd like to quickly verify its integration into the workbench.

**Comment by domhanak** [comment]
@yurloc  "Settings -> SSH Keystore" check kie-wb-distributions PR for wars

File: optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/pom.xml (hunk 1 above)
Comment (pefernan) [reply optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/pom.xml:176]: @yurloc I tried the tests and there were some weld unsatisfied dependencies error, I had to add the deps to solve that.

**Comment by yurloc** [comment]
@domhanak Thanks for the pointer. SSH Keystore is not visible on the Settings screen in this showcase. @pefernan can you check and fix the PR so that SSH Keystore is accessible?

`kie-wb-distributions` is not relevant for this PR.

**Comment by pefernan** [comment]
@yurloc it seems that optaplanner-wb isn't using the same component to set up the Admin page as the workbench, that might be the cause

**Comment by pefernan** [comment]
@yurloc added the ssh keystore into the admin page, you should be able to test it.

**Review by yurloc — APPROVED** [review]
Looks good. SSH keystore screen is accessible and key pair authentication works when cloning repositories.

**Review by ederign — APPROVED** [review]

**Pull request merged**